prac_09/unreliable_car_test.py

```diff
@@ -0,0 +1,22 @@
+"""
+CP1404/CP5632 Practical
+Test Unreliable Car
+"""
+from unreliable_car import UnreliableCar
+
+
+def main():
+    # Create a new unreliable car object with 50% reliability
+    car = UnreliableCar("Test Car", 100, 50)
+
+    # Attempt to test 10 cars
+    for i in range(10):
+        success = car.drive(1)
```

Comment:
Name implies that reliability of the car changes based on result, when reliability actually remains static at 50